rollup.config.js

```diff
@@ -0,0 +1,52 @@
+import path from "path";
+import alias from "rollup-plugin-alias";
+import commonjs from "rollup-plugin-commonjs";
+import nodeResolve from "rollup-plugin-node-resolve";
+import typescript from "rollup-plugin-typescript2";
+
+const mockPath = path.join(path.resolve("../../js/"), "mock_builtin");
```

Comment:
`path.resolve("../../js/")` uses the current working directory if I'm not mistaken. It would be better if you used __dirname here so it was relative to `rollup.config.js`

Reply:
Yes, that is what I need to do, plus also repath a couple other things since the travis structure is different.  Thanks.